src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.Win32.Unix.cs

```diff
@@ -68,6 +68,11 @@ internal static void GetDefaultIdentityAndHash(out object identity, out string h
             // by pulling directly from EntryAssembly.
 
             Assembly assembly = Assembly.GetEntryAssembly();
+
+            // Fall back on the executing assembly if needed
```

Comment:
How is this ever correct?  Won't this be the "System.IO.IsolatedStorage, ..."?  Maybe we discussed this before but I can't remember the reasoning which means we probably need a comment explaining.